check_coverage.bat

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+#Ubuntu
+#MIN_LINES_COV=74.9
+#MIN_FUNCTION_COV=86.5
```

Comment:
Why these lines are commented, but RHELs aren't?

Reply:
The rhel ones are lower thus we can use it. Currently onCommit tests rhel.